yt_dlp/extractor/daystar.py

```diff
@@ -0,0 +1,55 @@
+from .common import InfoExtractor
+from ..utils import js_to_json, urljoin
+
+
+class DaystarClipIE(InfoExtractor):
+    IE_NAME = 'daystar:clip'
+    _VALID_URL = r'https?:\/\/player\.daystar\.tv\/(?P<id>\w+)($|\W)'
+    _DOMAIN = 'https://www.lightcast.com/embed/'
+    _TEST = {
+        'url': 'https://player.daystar.tv/0MTO2ITM',
+        'info_dict': {
+            'id': '0MTO2ITM',
+            'ext': 'mp4',
+            'title': 'The Dark World of COVID Pt. 1 | Aaron Siri',
+            'description': 'Is the fear of the pandemic being used to suppress and erode your constitutional '
+                           'freedoms? Attorney Aaron Siri exposes the truth about what’s happening and how you can '
+                           'join the fight to preserve your liberties. (J2167)',
+            'thumbnail': r're:^https?://.+\.jpg',
+        },
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        webpage = self._download_webpage(url, video_id=video_id)
+
+        title = self._html_search_meta(['og:title', 'twitter:title'], webpage, fatal=True)
+        description = self._html_search_meta(['og:description', 'twitter:description'], webpage, fatal=True)
+
+        src_iframe = self._search_regex(r'\<iframe.*?src=\"(.*?)\"', webpage, 'src iframe')
```

Comment:
```suggestion
        src_iframe = self._search_regex(r'\<iframe[^>]+src="([^"]+)"', webpage, 'src iframe')
```
similar for other regexes. Non-greedy matching like this can cause catastrophic backtracking